README.md

```diff
@@ -62,6 +61,107 @@ In a separate directory invoke the generator via
 yo ibm-service-enablement
 ```
 
+## Adding new services
+
+To add a new services you will need to follow these steps.
+
+Create a new composed generated by adding a service folder under generators with the prefix service- then the service id of your new service. To illustrate, if service called S3 needs to be added the folder would be called service-s3.
+
+Under this service there is one folder named templates where you will store your templates per language. Currently, we support java-liberty, java-spring, node, python, and swift. If there is a language that you do not support make an issue.
```

Comment:
put folder names in ``. templates --> `templates/`

Typo: "If there is a language that you do not support" --> "If there is a language that WE do not support, create a new github issue."